### DISCUSSION TITLE
A Strange Problem

### DISCUSSION THREAD

**Discussion by Ada3181** [ExistentialAudio/BlackHole#657] [Q&A] [answered]
Hey! So I recently redownloaded Blackhole 16ch in hopes of recording episodes of a show I watch (commentary stuff). I set up the multi output on my mac and set up obs, but the episode wouldn't play. First I thought it was OBS, so I closed out of it. Then I realized it was the multi output device, since when I switched the output back to my headphones, the episode played perfectly. This was on a kisscartoon like site, so I tried the same thing on a Youtube video with the same results. The video freezes up, is unable to load, and it's only fixed if I don't have the output on multi output device. Has anyone else had this problem before? Is there any way of troubleshooting it? Here's how I have the multi output set up, in case anything looks wrong.
<img width="391" alt="Screen Shot 2022-09-06 at 9 04 47 PM" src="https://user-images.githubusercontent.com/96184040/188772601-7b53ca5c-7b13-4bc2-817d-6427fe5f8e48.png">
I also tested this on two different browsers, Firefox and Safari to see if one performed better, but both didn't show the video through the multi output.
I'm happy to answer any additional questions. I really hope I can fix this soon. Thanks much!

**Comment by devinroth** [comment, marked as answer]
https://github.com/ExistentialAudio/BlackHole/wiki/Multi-Output-Device#4-select-output-devices

Devin

**Reply by Ada3181** [reply]
Well gracious me I'm dumb. Thank you very much for the link.
